app/airq/models/users.py

```diff
@@ -0,0 +1,23 @@
+from flask_login import UserMixin
+from werkzeug.security import check_password_hash
+from werkzeug.security import generate_password_hash
```

Comment:
Why did you choose werkzeug? Just curious

Reply:
It's the default for Flask. I hate that flask has so many dependencies and you actually end up needing most of them.